### PR TITLE
updated all page titles and meta descriptions

### PR DESCRIPTION
**Pull request by wmschott**
Ready for review

### CODE REVIEW

**Comment by wmschott** [comment]
@bbewan Typo's have all been fixed accept 2.  It is flagging the word analytics which seems to be spelled correctly.  Not sure how to fix?

File: troubleshooting/ios/missing_schemes.adoc [hunk 1]
```diff
@@ -1,3 +1,9 @@
+--- 
+titletext: How to trouble shoot missing schemes during iOS CI build
+description: >
+  Not seeing your schema? It is likely because the target of the app in
```
Comment (bbewan) [inline troubleshooting/ios/missing_schemes.adoc]: Buddybuild treats every target as a separate app, and a scheme ties together a target with a build configuration. This section describes how to ensure that you are not missing any schemes.

File: troubleshooting/ios/spec_repo_not_compatible_with_older_cocoapods_versions.adoc [hunk 2]
```diff
@@ -1,3 +1,9 @@
+--- 
+titletext: Trouble shooting builds using 0.x versions of CocoaPods
```
Comment (bbewan) [inline troubleshooting/ios/spec_repo_not_compatible_with_older_cocoapods_versions.adoc]: How to solve build problems with CocoaPods 0.x

File: troubleshooting/ios/spec_repo_not_compatible_with_older_cocoapods_versions.adoc [hunk 3]
```diff
@@ -1,3 +1,9 @@
...
+description: >
+  On Nov 11, 2016, CocoaPods introduced a breaking change that affects customers
+  using versions 0.x. Consider upgrade to CocoaPods 1.x.
```
Comment (bbewan) [inline troubleshooting/ios/spec_repo_not_compatible_with_older_cocoapods_versions.adoc]: ... Consider upgrading...

File: troubleshooting/user_not_getting_alert_when_a_new_version_of_app_is_available.adoc [hunk 4]
```diff
@@ -1,3 +1,9 @@
+--- 
+titletext: Mobile testers not getting new app version alert
```
Comment (bbewan) [inline troubleshooting/user_not_getting_alert_when_a_new_version_of_app_is_available.adoc]: How to ensure that mobile testers get new version alerts

File: troubleshooting/user_not_getting_alert_when_a_new_version_of_app_is_available.adoc [hunk 5]
```diff
@@ -1,3 +1,9 @@
...
+description: >
+  Buddybuild's SDK is used alert mobile app testers when a new build is
```
Comment (bbewan) [inline troubleshooting/user_not_getting_alert_when_a_new_version_of_app_is_available.adoc]: The buddybuild SDK alerts mobile...

File: troubleshooting/user_not_getting_alert_when_a_new_version_of_app_is_available.adoc [hunk 6]
```diff
@@ -1,3 +1,9 @@
...
+  available. To enable this feature the following configurations are required.
```
Comment (bbewan) [inline troubleshooting/user_not_getting_alert_when_a_new_version_of_app_is_available.adoc]: Drop everything after `available.`

File: billing/transfer_apps.adoc [hunk 7]
```diff
@@ -1,3 +1,9 @@
+--- 
+titletext: Transfer buddybuild apps between organizations
+description: >
+  You can transfer an app from one organization to another. The app will inherit
+  the feature set available in the new organization.
```
[the remark this answers is not in the document]
Comment (Palleas) [reply billing/transfer_apps.adoc]: I think @wmschott was talking about those limitations. Do you have an alternative to "feature set"?

[the remark this answers is not in the document]
Comment (bbewan) [reply billing/transfer_apps.adoc]: Perhaps:
```
When you transfer an app to another organization, it adopts the features and limitations of the new organization's plan.
```

File: deployments/ios/code_signing/upload_manually.adoc [hunk 8]
```diff
@@ -1,3 +1,10 @@
+--- 
+titletext: How to Upload iOS Code Signing Certificates Manually
+description: >
+  Buddybuild provides 2 ways to manage Code Signing. The easy way: automatic
```
[the remark this answers is not in the document]
Comment (Palleas) [reply deployments/ios/code_signing/upload_manually.adoc]: Good catch, ping @wmschott ?

File: quickstart/ssh.adoc [hunk 9]
```diff
@@ -1,3 +1,9 @@
+--- 
+titletext: Mobile CI/CD repositories using SSH integration
+description: >
+  If your repository lives in Beanstalk or your own source control system, use
```
[the remark this answers is not in the document]
Comment (Palleas) [reply quickstart/ssh.adoc]: That's only true if the user set up the webhook.

[the remark this answers is not in the document]
Comment (bbewan) [reply quickstart/ssh.adoc]: Right you are. Perhaps:
```
SSH is the simplest way for buddybuild to access your repository and start building your app.
```

**Comment by bbdocs** [comment]
See the website here: http://91-bb-docs.surge.sh

**Comment by bbdocs** [comment]
See the website here: http://91-bb-docs.surge.sh

**Pull request merged**